nms/packages/magmalte/fbc_js_core/ui/hooks/useRouter.js

```diff
@@ -24,16 +24,31 @@ import {__RouterContext as RouterContext} from 'react-router-dom';
 export const useRelativeUrl = () => {
   const {url} = useRouteMatch();
   return useCallback(
-    (path: string) => (url === '/' && path ? path : `${url}${path}`),
+    (path: string) => {
+      if (!path) {
+        return url;
+      }
+      if (path[0] !== '/') {
+        path = `/${path}`
+      }
```

Comment:
Mostly just out of curiosity - why add this permissiveness?

Reply:
Just as safety net. We should still try to use it consistently until it gets removed by upgrading the react-router :)